src/main/java/com/faforever/client/query/RangeFilterController.java

```diff
@@ -44,17 +51,20 @@ public void initialize() {
     valueTransform = (value) -> value;
   }
 
+  @SneakyThrows
   public Optional<List<Condition>> getCondition() {
     List<Condition> conditions = new ArrayList<>();
-    if (!lowValue.getText().isBlank() && rangeSlider.getLowValue() > rangeSlider.getMin()) {
+    double lowValueValue = numberFormat.parse(numberFormat.format(rangeSlider.getLowValue())).doubleValue();
```

Comment:
it's normalizing value to desired format. why sent different value to what is displayed (ignoring value transformer :))

Reply:
But the low value is set using the formatted value right. So outside double precision issues it should already be the same

Reply:
well I was developing most of the time without snapToTicks true and without this code, it looked like this
![image](https://github.com/FAForever/downlords-faf-client/assets/8619173/6b8ddf86-6f5e-4e55-bed6-618f637f3d01)
with snapToTicks true it will align after you release slider.
but I am not sure that we want to rely on that option.

Reply:
I would say that it was developed so that the range slider has the true value and the text is just for informing the user really

Reply:
which text?

Reply:
The min an max text boxes